public/themes/lara-light-indigo/theme.css

```diff
@@ -2761,7 +2761,7 @@
   outline-offset: -0.15rem;
 }
 .p-datatable.p-datatable-hoverable-rows .p-datatable-tbody > tr:not(.p-highlight):not(.p-datatable-emptymessage):hover {
-  background: #e9ecef;
+  background:	var(--teal-100);
```

Comment:
Why was this change made? Doesn't seem bad, just trying to learn more about front-end dev

Reply:
The change is to reflect Kate's design, when hovering over a row, the color was light grey. This made it a light teal that is consistent with Seattle humane color pallet and stands out a bit more than the original set color by primereact

Reply:
this adds consistency to our colors because teal-100 is part of prime reacts color palettes, we probably need to diverge from it at certain parts but its good to try and use theme colors as much as possible